data_model.md

```diff
@@ -134,6 +134,10 @@ that enable power forecasts. These parameters will be listed under a
         in units of 1/C. Typically -0.002 to -0.005 per degree C.
     -   *Tracking type* - type of tracking system, i.e. fixed, single axis,
         two axis
+    -   *Irradiance loss factor* - applied to plane of array irradiance after
+        reflection losses but before spectral mismatch losses. Default 0%.
```

Comment:
Implies that the model sequence will include a reflection and a spectral mismatch model (even a constant 1.0).  Will these models be used as a rule?  If not, then the `after...before` phrase needs a condition. `Applied to plane of array irradiance. If reflection and spectral loss models are used, applies after reflection losses but before spectral mismatch losses.`

Reply:
Otherwise, yes.

Reply:
Good question. The metadata here serves two purposes: 
1. provide forecasters with basic information about the system they are supposed to forecast. The free-form extra parameters section is available for anything else. 
2. provide the internal benchmark forecasts with basic system info to be consumed by pvlib functions.

For 1, whether or not the models are used is out of our control. For 2, I think we should include default reflection losses that are independent of module type, but I'm less sure about spectral mismatch losses.

I think we can use your wording and leave it at that.

Or we could retreat and say we're just not going to support poorly defined loss factors in the standard metadata and if you want something more then put it in extra parameters and let the forecasters sort it out.

Reply:
Or drop irradiance and keep DC/AC? Probably not many people that would use irradiance loss factor in a meaningful way in this platform.

Reply:
Agree. Let's leave it out for now, add it later if enough users request it.  Besides, a constant scalar applied to irradiance has the same effect as a scalar applied to DC current.